distribution/docs/src/main/jdocs/content/_plugins/resource-usage-plugin-contents.adoc

```diff
@@ -0,0 +1,24 @@
+:type: plugin
+:status: published
+:title: Resource Usage Plugin
+:link: _resource_usage_plugin
+:plugintypes: preresource,postquery,postresource
+:summary: Monitors and limits data usages.
+
+The Resource Usage Plugin monitors and limits data usage, and enables monitoring and cancelling running queries.
```

Comment:
`monitoring` and `cancelling` are what should be parallel. I did decide to drop `monitoring` anyway, because it's mentioned in the first part of the sentence.